cubeviz/tests/test_image_viewer.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+def test_stats_box(cubeviz_layout):
+    cl_viewer = cubeviz_layout.split_views[1]._widget
+
+    if cl_viewer._subset is not None:
```

Comment:
What is the purpose of the `if`/`else` here? This test only runs once and the state of the application should always be the same when the test starts, so only one of these branches is ever going to be executed.

Reply:
```
    def update_stats(self):

        if self._subset is not None:
            self.show_roi_stats(self.current_component_id, self._subset)
        else:
            self.show_slice_stats()
```
This is the code block from image_viewer.py, which uses that same if / else to determine which version of `show_stats` to use. I decided to mimic that in case that test data set changes, but I can remove it if that makes more sense.

Reply:
I think what we actually want to do is to test both of those code paths in `update_stats` in separate tests, and not have any branching logic in either of the tests. This will require creating (and deleting) a subset in one of the tests. There should be an example of that in at least one of the other tests.